front-end-side/src/Pages/Friends.js

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import Navbar from "../../src/Components/Navbar";
+import "../Pages/Friends.css";
 const Friends = () => {
     return (
         <div>
```

Comment:
Change it to <div class="container"> in order to align content right

Reply:
I want it alligned to the left.

Reply:
It has to have this allignment 🤦‍♂️